shared/oae/js/jquery-plugins/jquery.infinitescroll.js

```diff
@@ -274,7 +274,20 @@ define(['jquery', 'underscore', 'oae.api.util', 'oae.api.i18n', 'oae.api.l10n'],
                 oaeL10n.timeAgo($listContainer);
 
                 // Apply multi-line threedotting to the tile titles
-                $('.oae-tile h3').dotdotdot({'watch': 'window'});
+                $('.oae-tile h3').dotdotdot({
+                    'callback': function(isTruncated, orgContent) {
```

Comment:
I'd prefer to not have inconsistent tooltip behaviour within the same list. I'd rather sacrifice some accessibility here and always have the title attribute for the list items.